pymatbridge/messenger/tests/test_env.py

```diff
@@ -0,0 +1,33 @@
+import numpy.testing as npt
+import sys
+import os
+import pkg_resources
+
+from pymatbridge import messenger
+
+ROOT = __file__
+CONFIG = os.path.realpath(os.path.join(ROOT, '../../../../config.ini'))
```

Comment:
Yes, so after moving the messenger directory into the pymatbridge folder, I had to make this as a temporary fix just to make sure these tests still run.

Ultimately since now we run tests as `python setup.py test`, I need to do some refactoring using `pkg_resources` to grab the `config.ini` rather than anchoring to the  `__file__`.